ocean_lib/ocean/ocean.py

```diff
@@ -141,20 +144,37 @@ def create_data_token(
             datatoken = ocean.create_data_token("Dataset name", "dtsymbol", from_wallet=wallet)
         ```
 
-        :param name: Datatoken name, str
-        :param symbol: Datatoken symbol, str
+        :param name: ERC721 token name, str
+        :param symbol: ERC721 token symbol, str
         :param from_wallet: wallet instance, wallet
-        :param cap: Amount of data tokens to create, denoted in wei, int
+        :param template_index: Template type of the token, int
+        :param additional_erc20_deployer: Address of another ERC20 deployer, str
+        :param token_uri: URL for ERC721 token, str
 
-        :return: `Datatoken` instance
+        :return: `ERC721Token` instance
         """
 
-        dtfactory = self.get_dtfactory()
-        tx_id = dtfactory.createToken(blob, name, symbol, cap, from_wallet=from_wallet)
-        address = dtfactory.get_token_address(tx_id)
-        assert address, "new datatoken has no address"
-        dt = DataToken(self.web3, address)
-        return dt
+        erc721_factory = self.get_erc721_factory()
```

Comment:
Let's get all modifications to Ocean class merged first. I'll copy the docstrings from your branch.

Reply:
Sure